frontend/app/src/components/history/transactions/TransactionContent.vue

```diff
@@ -350,7 +396,19 @@ const getId = (item: EthTransactionEntry) => item.txHash;
 
 const selected: Ref<EthTransactionEntry[]> = ref([]);
 
-const { filters, matchers, updateFilter } = useTransactionFilter();
+const readOnlyFilters: ComputedRef<Partial<Filters>> = computed(() => {
+  const filters: Partial<Filters> = {};
+  const protocolVal = get(protocol);
+  if (protocolVal) filters.protocols = [protocolVal];
+
+  const eventTypeVal = get(eventType);
+  if (eventTypeVal) filters.eventTypes = [eventTypeVal];
+
+  return filters;
+});
+
+const { filters, matchers, updateFilter } =
+  useTransactionFilter(readOnlyFilters);
```

Comment:
See the comment from `useTransactionFilter` and let me know what you think about it.